x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/lookup/EnrichLookupIT.java

```diff
@@ -52,6 +53,7 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
+@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/99826")
```

Comment:
I hate disabling this. `ENRICH` seems to work, but the IT isn't. It could be that we simply don't test the enrich path I'm breaking with this PR though.

Reply:
You can leave it muted. I will integrate BlockFactory to the enrich operator.

Reply:
thanks. I was digging around and learning a lot. But it wasn't quick.